packages/sharing-common/src/proto/models.proto

```diff
@@ -4,6 +4,7 @@ message AppData {
   string entrypoint = 1;
   map<string, File> files = 2;
   repeated string requirements = 3;
+  optional bool languageServer = 4;
```

Comment:
So this change should be reverted as `languageServer` is always set true.

Reply:
See my comment above why I made this optional

Reply:
Reverted back this change